apps/sms/test/unit/thread_ui_test.js

```diff
@@ -5925,7 +5942,9 @@ suite('thread_ui.js >', function() {
         MessageManager.on.withArgs('message-received').yield({
           message: testMessage
         });
-      }).then(done, done);
+
+        return Contacts.findByPhoneNumber.lastCall.returnValue;
```

Comment:
but I'm quite sure I saw it broken without it, so I'll leave it and add some comment; maybe it would be an intermittent failure without this.